src/styles/blocks/rating.scss

```diff
@@ -0,0 +1,13 @@
+.rating {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding-inline: $standard-inline-pad;
```

Comment:
Ensure that the variable `$standard-inline-pad` is correctly spelled and consistent with the naming conventions.